openprescribing/frontend/feedback.py

```diff
@@ -0,0 +1,27 @@
+from django.conf import settings
+from django.core.mail import get_connection, EmailMultiAlternatives
+from django.template.loader import get_template
+
+
+def send_feedback_mail(user_name, user_email_addr, subject, message, url):
+    connection = get_connection()
+
+    ctx = {
+        'name': user_name,
+        'url': url,
+        'message': message,
+    }
+
+    template = get_template('feedback_email.txt')
+    body = template.render(ctx)
+
+    email = EmailMultiAlternatives(
+        subject='OpenPrescribing Feedback: {}'.format(subject),
+        body=body,
+        from_email='{} <{}>'.format(user_name, user_email_addr),
```

Comment:
Is this `from_email` right?  (I know, I wrote it...)

As we're setting `reply_to`, any replies to the email will go to the right place.

And we want to include the sender's name in `from_email`, so that the name shows up in the right place in the inbox UI.

But is using `user_email_addr` the right thing to do here?  We send email through Mailgun, and in the settings, we have:

    ANYMAIL["MAILGUN_SENDER_DOMAIN"] = "openprescribing.net"

I think that means we should be sending from an "@openprescribing.net" address.  @evansd do you agree?

Reply:
This is a good point. I think it will work in the sense that the mailgun won't reject the mail (`sender` and `from` don't need to match), but it may not help our spam issues.

Would an alternative be to use just their name in the `from` field, and then include their full email address in the body of the email for searchability?

Reply:
Yes, there are definitely spam considerations:


```
Sender: <peter.inglesby=gmail.com@openprescribing.net>
Message-ID: <20181010183214.1.BC2175AA63CDED91@openprescribing.net>
Reply-To: <peter.inglesby@gmail.com>
To: <ebmdatalab@phc.ox.ac.uk>
From: Peter Inglesby <peter.inglesby@gmail.com>
Subject: Test 4
Content-Type: text/plain; charset="ascii"
MIME-Version: 1.0
X-NTG-DKIM-verify: pass 
X-Oxmail-Spam-Status: score=0.5 tests=FREEMAIL_FORGED_FROMDOMAIN,FREEMAIL_FROM,HEADER_FROM_DIFFERENT_DOMAINS
  0.0 FREEMAIL_FROM          Sender email is commonly abused enduser mail provider
                             (peter.inglesby[at]gmail.com)
  0.2 HEADER_FROM_DIFFERENT_DOMAINS From and EnvelopeFrom 2nd level mail
                             domains are different
  0.2 FREEMAIL_FORGED_FROMDOMAIN 2nd level domains in From and EnvelopeFrom
                              freemail headers are different
```

And note how the sender email address has been changed.  (I'm not sure what's done that.)

> Would an alternative be to use just their name in the from field, and then include their full email address in the body of the email for searchability?

Good idea, I'll do that.